app/helpers/application_helper.rb

```diff
@@ -114,6 +114,15 @@ def valid_url?(input)
     false
   end
 
+  def url_size(url)
```

Comment:
Can we move this to a private method in the `rooms_controller` instead?

Reply:
I would also replace this with something less general.

```
  def valid_avatar?(url)
    return false if URI.regexp.match(url).nil?
    uri = URI(url)
    http = Net::HTTP.new(uri.host, uri.port)
    http.use_ssl = true if uri.scheme == 'https'
    response = http.request_head(uri)
    return response['content-length'].to_i < Rails.configuration.max_avatar_size
  end
```

Reply:
Also, avoid accessing the `ENV` variable directly and set it in `config/application.rb` instead